client/src/partials/patients/search/search.js

```diff
@@ -0,0 +1,65 @@
+angular.module('bhima.controllers')
+.controller('PatientSearchController', PatientSearchController);
+
+PatientSearchController.$inject = [
+  '$translate', 'PatientService', 'DateService'
+];
+
+/** 
+ * Users and Permission Controller 
```

Comment:
Ce module n'est pas le "Users and Permissions Controller".  S'il vous plait, soyez vigilant quand vous copiez et collez code.